views/tables/StudiesTable/StudiesTable.test.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+
+import { render, screen, within } from '@testing-library/react'
+
+import StudiesTable from '.'
+import { SITE_NAMES } from '../../../server/utils/siteNames'
+
+describe(StudiesTable, () => {
+  const studies = [
+    {
+      daysInStudy: 655,
+      numOfParticipants: 2,
+      study: 'LA',
+      updatedAt: '2024-01-05T06:00:00.000Z',
+    },
```

Comment:
should we have a fixture to create a subject day data object?

Reply:
[b6abb0b](https://github.com/AMP-SCZ/dpdash/pull/723/commits/b6abb0bd1e5a49faa249a852a52ea7280c83be87)

Reply:
[3fb9be8](https://github.com/AMP-SCZ/dpdash/pull/723/commits/3fb9be899eba72e300d84557422ec5fa4d84500d)